andela/mervels-bn-frontend
# #168131532 User Role Settings UI

#### What does this PR do?
Implements assign user roles
#### Description of Task to be completed?
- Super admin should be able to assign user roles
- implement error handling for failed delivery
#### How should this be manually tested?
- Checkout to the branch `ft-user-role-setting-168131532`
- run npm npm install
- Run npm start
- Ensure that you are signed in as a super admin
- Head over the route `/assign-role`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131531](https://www.pivotaltracker.com/story/show/168131532)
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-11-21 at 16 29 15" src="https://user-images.githubusercontent.com/26383328/69346815-1e2d8880-0c7c-11ea-84f5-4565d40c67df.png">
#### Questions:
N/A
